packages/desktop-client/src/components/reports/ReportOptions.ts

```diff
@@ -16,11 +18,11 @@ export const defaultReport: CustomReportEntity = {
   startDate,
   endDate,
   isDateStatic: false,
-  dateRange: 'Last 6 months',
+  dateRange: t('Last 6 months'),
   mode: 'total',
   groupBy: 'Category',
-  interval: 'Monthly',
-  balanceType: 'Payment',
+  interval: t('Monthly'),
+  balanceType: t('Payment'),
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏should these be translated? I'm not 100% sure. If they are saved in the DB, then I think they shouldn't be translated.

cc @carkom

Reply:
They are saved in the DB in English. The app creates the report based on the values being in English.

Reply:
> They are saved in the DB in English. The app creates the report based on the values being in English.

Alright, thanks. @MatissJanis I'll correct it in the evening.